lib/urbanopt/geojson/building.rb

```diff
@@ -107,92 +106,81 @@ def create_building(create_method, model, origin_lat_lon, runner, zoning=false,
         end
         return spaces
       end
-
+      alias_method :create_other_building, :create_building
       ##
-      # This method is used to create the surrounding buildings as shading objects.
-      #
-      # Returns an array of instances of +OpenStudio::Model::Space+ .
+      # Return the features multi polygon in an array of the form coordinate pairs in double nested Array
       #
       # [Parameters]
-      # * +other_building_type+ - _Type:String_ - Describes the surrounding buildings. Currently 'ShadingOnly' is the only option that is processed.
-      # * +other_buildings+ - _Type:URBANopt::GeoJSON::FeatureCollection_ - List of surrounding buildings to include (self will be ignored if present in list).
-      # * +model+ - _Type:OpenStudio::Model::Model_ - An instance of an OpenStudio Model.
       # * +origin_lat_lon+ - _Type:String_ - An instance of +OpenStudio::PointLatLon+ indicating the latitude and longitude of the origin.
       # * +runner+ - _Type:String_ - An instance of +Openstudio::Measure::OSRunner+ for the measure run.
-      def create_other_buildings(other_building_type, other_buildings, model, origin_lat_lon, runner, zoning=false)
-        feature_id = @feature_json[:properties][:id]
-        # Nearby buildings to be converted to shading.
-        convert_to_shades = []
-
-        if other_buildings[:features].nil?
-          runner.registerWarning("No features found in #{other_buildings}")
-          return []
-        end
-
-        building_points = []
-        multi_polygons = get_multi_polygons
+      # * +zoning+ - _Type:Boolean_ - Should be true if you'd like to utilize aspects of function that are specific to zoning.
+      def feature_points(origin_lat_lon, runner, zoning)
```

Comment:
can we make origin_lat_lon, runner, and zoning instance variables on the Building class?